styles.css

```diff
@@ -0,0 +1,69 @@
+body {
+  min-width: 375px;
+  margin: 0;
+  padding: 0;
+
+  /* box-sizing: border-box; */
+}
+
+.menu {
+  width: 1.25rem;
+  margin-left: 17.75rem;
+  margin-top: 1.188rem;
+}
+
+header {
+  background-image: url(./icons/image_geometry_1.png);
+  background-repeat: no-repeat;
+  height: 4.625rem;
+  display: flex;
+  align-items: center;
+}
+
+.homepage {
+  width: 20rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+h1 {
+  font-family: Playfair Display, arial, sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+}
+
```

Comment:
- [ ] Great job on implementing the first milestone of your portfolio project as specified in the project requirement! 👏 However, there are some minor issues 🐞 that we need to address and fix before you move on to the next milestone. 
   - [ ] Kindly consider adjusting the `margin` you have below the `h1` tag as it's too much and doesn't match that of the `Figma` design. See below for more details:
   <table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="316" alt="Screenshot 2023-07-05 at 19 10 28" src="https://github.com/JohnteFresh/IrunguJohnson-Mobile-Portfolio/assets/107192076/a02d5e86-c01a-4128-890c-3c0a9c4c9716">
</td>
<td>
<img width="328" alt="Screenshot 2023-07-05 at 19 09 06" src="https://github.com/JohnteFresh/IrunguJohnson-Mobile-Portfolio/assets/107192076/a48d8e40-1165-4525-9ff6-6ee39f6a701c">
</td>
</tr>
</table>